py/sentry_streams/user_functions/sample_filter.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+
+class EventsPipelineFilterFunctions:
```

Comment:
While we're already adding the filter functions to `user_functions` directory, can you move the `sample_map` functions into the same directory?

Then `user_functions` can be a central place (for now) where users define their custom logic.